README.md

```diff
@@ -30,13 +32,17 @@ $ UPDATED_DOMAINS="example.org,more.example.org"
 $ docker exec -it apache-ssl /run_letsencrypt.sh --domains $UPDATED_DOMAINS
 ```
 
+
 ### Configuring docker-container
 
 It's possible to configure the docker-container by setting the following environment-variables at container-startup:
 
 - `DOMAINS`, configures which for which domains a SSL-certificate shall be requested from Let's Encrypt, default is `""`. Must be given as comma-seperated list, f.e.: `"example.com,my-internet.org,more.example.com"`.
 - `WEBMASTER_MAIL`, Let's Encrypt needs a mail-address from the webmaster of the requested domain. You have to set it, otherwise Let's Encrypt won't give the certificates. Default is `""`. Must be given as simple mail-address, f.e.: `"webmaster@example.com"`.
-- `STAGING`, if set with a not-null string use Let's Encrypt Staging environment to avoid rate limits during development.
+- `STAGE`,
```

Comment:
Thanks for refining this!
Would you add also the option `production` with some comment?